packages/widgets/src/components/TableBots/types.ts

```diff
@@ -0,0 +1,48 @@
+import { Interpolation, Theme } from '@emotion/react';
+import { ExchangeType } from './stories/TableBots/components/Exchange/types';
+import { TradeType } from './stories/TableBots/components/Trade/types';
+
+export interface PNL {
+  amount: number;
+  currency: string;
+  isIncreasing: boolean;
+}
+
+export interface ROI {
+  amount: number;
+  isIncreasing: boolean;
+}
+
+export interface TableBotItem {
```

Comment:
Should this interface and `NoControlOrdersItem` be here or deeper on the stories level?

Reply:
Agree, moved